src/components/case_manager/PersonTable.tsx

```diff
@@ -289,7 +289,14 @@ export default function EnhancedTable<
   return (
     <Box sx={{ width: "75%" }}>
       {/* Form can't be integrated with table (can't span multiple cells), so the form is on the outside with inputs referencing its id */}
-      <form onSubmit={onSubmit} id="table_input_form"></form>
+      <form
+        onSubmit={(event: React.FormEvent<HTMLFormElement>) => {
+          onSubmit(event);
```

Comment:
type for `onSubmit` should be updated above to return `Promise<void> | void` and then we can await it here to prevent resetting the form if `onSubmit` is async and throws